src/views/components/AnalyticsGraph/index.vue

```diff
@@ -0,0 +1,312 @@
+<template>
+  <div :id="this.title">
+    <svg width="80vw" height='33vh'></svg>
+  </div>
+</template>
+<script>
+import * as d3 from "d3";
+export default {
+  name: 'graph',
+  data () {
+    return {
+    };
+  },
+  props : ['values' , 'title', 'xUnits', 'yUnits'],
+
+  mounted() {
+    /**@description Create an event listener for window resizing that rerenders the graph */
+    this.$nextTick(function() {
+      window.addEventListener('resize', this.redrawGraph);
+    })
+    //initialize the graph
+      this.drawLinesGraph(document.getElementById(this.title).offsetHeight * 0.90, document.getElementById(this.title).offsetWidth * 0.85, 
```

Comment:
just out of curiosity, what does 0.9 and 0.85 represent? maybe a good idea to have a constant for these so they're not "magic numbers" :P

Reply:
They are the most magical numbers to make the graph dimensions so perfect. I'll make them constants